completions/Makefile.am

```diff
@@ -797,7 +797,10 @@ CLEANFILES = \
 
 symlinks: $(DATA)
 	$(ss) 7z \
-		7za
+		7za \
+		7zr \
+		7zz \
+		7zzs
```

Comment:
We should add these to `CLEANFILES` and `.gitignore` too, git grep for 7za to find the spots.

Reply:
Good catch!  Thanks @scop.  Fix pushed.